docker/install-dynamic-plugins.py

```diff
@@ -95,8 +137,30 @@ def main():
                 file.extract(member, path=directory)
             elif member.isdir():
                 print('\t\tSkipping directory entry', member.name, flush=True)
+            elif member.islnk() or member.issym():
+                if not member.linkpath.startswith('package/'):
+                  raise InstallException('NPM package archive contains a link outside of the archive: ' + member.name + ' -> ' + member.linkpath)
+
+                member.name = member.name[8:]
```

Comment:
Minor nit, it's considered best practice to not reassign a variable that iterates. If possible, we should create a new variable instead. 

```suggestion
                name = member.name[8:]
```

Reply:
Here precisely want to change the name field of the `member` variable (remove the `package` suffix) , so that the member is extracted with the right name by the `file.extract(member, path=directory)` call.

Reply:
Oh, I must've missed that.